invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v7.0.0.json

```diff
@@ -0,0 +1,1480 @@
+                  }
+                }
+              },
+              "scheme": {
+                "type": "keyword"
+              }
+            }
+          },
+          "resource_type": {
+            "type": "object",
+            "properties": {
+              "@v": {
+                "type": "keyword"
+              },
+              "id": {
+                "type": "keyword"
+              },
+              "title": {
+                "type": "object",
+                "dynamic": "true"
+              },
+              "props": {
+                "type": "object",
+                "properties": {
+                  "type": {
+                    "type": "keyword"
+                  },
+                  "subtype": {
+                    "type": "keyword"
+                  }
+                }
+              }
+            }
+          },
+          "rights": {
+            "type": "object",
+            "properties": {
+              "@v": {
+                "type": "keyword"
+              },
+              "id": {
+                "type": "keyword"
+              },
+              "title": {
+                "type": "object",
+                "dynamic": "true"
+              },
+              "description": {
+                "type": "object",
+                "dynamic": "true"
+              },
+              "props": {
+                "type": "object",
+                "properties": {
+                  "url": {
+                    "type": "keyword"
+                  },
+                  "scheme": {
+                    "type": "keyword"
+                  }
+                }
+              },
+              "link": {
+                "type": "keyword",
+                "index": false
+              },
+              "icon": {
+                "type": "keyword",
+                "index": false
+              }
+            }
+          },
+          "sizes": {
+            "type": "keyword",
+            "ignore_above": 256
+          },
+          "subjects": {
+            "type": "object",
+            "properties": {
+              "@v": {
+                "type": "keyword"
+              },
+              "id": {
+                "type": "keyword"
+              },
+              "subject": {
+                "type": "text",
+                "fields": {
+                  "keyword": {
+                    "type": "keyword"
+                  }
+                }
+              },
+              "scheme": {
+                "type": "keyword"
+              }
+            }
+          },
+          "combined_subjects": {
+            "type": "keyword"
+          },
+          "title": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256,
+                "normalizer": "custom_normalizer"
+              }
+            },
+            "analyzer": "custom_analyzer",
+            "search_analyzer": "custom_analyzer"
```

Comment:
Comment: Should this also be applied to other fields such as additional titles, ... ?

Reply:
Added more fields!